src/components/commit-list/CommitList.js

```diff
@@ -17,13 +18,16 @@ const Commit = ({ sha, commit, commiter, author, url }) => {
       {commiter && (
         <Text fontSize="14px">Commited by {commit.commiter.name}</Text>
       )}
-      <Text fontSize="14px">Commited on {commit.author.date}</Text>
+      <Text fontSize="14px">
+        Commited on{' '}
```

Comment:
Do you think it could look cleaner without the "commited on"? I think having just the date could make sense

Reply:
hmm, i think in terms of UI, some terminology of commits makes more sense than just the date.

we dont really say in other places in the page that the list on the screen is the list of commits.

I took the idea from github

<img width="415" alt="Screen Shot 2021-01-12 at 10 11 24 AM" src="https://user-images.githubusercontent.com/20917792/104333015-c58fff00-54be-11eb-94ef-360054080664.png">

Reply:
Our "committed on" line is on its own line, unlike this one. It's just more text to parse, making it harder to quickly scan the dates :)